stylelint.config.js

```diff
@@ -14,6 +14,9 @@ module.exports = {
 		"@spectrum-tools/stylelint-theme-alignment",
 		// "stylelint-high-performance-animation",
 	],
+	ignoreFiles: [
+		"tools/bundle/dist/*.css",
+	],
```

Comment:
If I'm reading this correctly, I think we should be using _either_ the .stylelintignore file _or_ the `ignoreFiles` property in the config. I'm flexible with whichever approach we use but I'd like to see all the same data in it. Would you mind either migrating the content from the stylelintignore to here or moving this out to the stylelintignore file?

https://stylelint.io/user-guide/ignore-code#files-entirely

Reply:
I'm happy to include it but I feel like the ignoreFiles list should match what's in .stylelintignore and/or we just remove .stylelintignore and let this be the source of truth

Reply:
I've added to the `ignoreFiles` list to mirror what's in the .stylelintignore. The syntax had to change a bit to use the glob patterns and also apply for some of the linting results that were one directory up.